install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1900,6 +1900,12 @@ envoy:
   # -- DNS policy for Cilium envoy pods.
   # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
   dnsPolicy: ~
+  debug:
+    admin:
+      # -- Enable admin interface for cilium-envoy.
```

Comment:
should we enhance the documentation that the port is only bound on the loopback ifaces and that access is meant via `kubectl port-forward`?